### PR TITLE
ui updates

### PR DESCRIPTION
**Pull request by yangevelyn**

### CODE REVIEW

File: webapp/status_server.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ def update_resource(ind, val=10):
   for resource in RESOURCES:
     resource_dict[resource] = get_resource(resource) 
 
+  print(resource_dict)
```
Comment (labib666) [inline webapp/status_server.py]: Is this required?

File: webapp/templates/base.html [hunk 2]
```diff
@@ -40,30 +40,62 @@
               var button = `<button type="button" class="btn btn-light" onclick="completeTask('${key}')">Done</button>`
 
               if(!val.completed){
+                // $(active).append(`
```
Comment (labib666) [inline webapp/templates/base.html:43]: Unless likely to be used in the future, it's nice to get rid of dead code.

**Review by labib666 — APPROVED** [review]
LGTM!

**Review by shaynemei — APPROVED** [review]
LGTM!

File: webapp/status_server.py (hunk 1 above)
Comment (yangevelyn) [reply webapp/status_server.py]: No, good catch thanks!

**Pull request merged**